### ISSUE TITLE
Broken link in the UXP Toolchain docs

### ISSUE THREAD

**Issue by ttmc** [AdobeDocs/uxp-photoshop#127]
The UXP Toolchain docs page at https://www.stage.adobe.io/photoshop/uxp/guides/uxp_guide/uxp-toolchain includes the sentence, "Here is a step-by-step walkthrough showing the creation of a Photoshop UXP plugin." That sentence is hyperlinked to the following URL:

https://www.stage.adobe.io/photoshop/uxp/udt-walkthrough/

When I go to that page, it says, among other things, "HttpStatusCode: 404" and "ErrorCode: WebContentNotFound". I did a search and I think the page in question got moved to:

https://www.stage.adobe.io/photoshop/uxp/guides/udt-walkthrough/

**Comment by chuckweger** [comment]
Hello @ttmc , thanks for pointing this out. The link is correct on the production site, here:

https://www.adobe.io/photoshop/uxp/guides/uxp-developer-tool/

May I ask how you found the staging site? Many things on that site are obsolete or in progress, and should not be relied upon. It shouldn't be showing up in search engines.

Regards,
Chuck

**Comment by ttmc** [comment]
I wondered about the word "stage" in the subdomain, but I figured that surely it must mean something other than staging...

I got there by following links from https://www.stage.adobe.io/photoshop/uxp/ --- but I don't remember where I got that URL. I probably got it and bookmarked it sometime around Adobe MAX, and I've been using it as a starting point ever since.

Doh!

I will update my bookmarks. Feel free to close this issue.
